packages/react/src/ClientOnly.tsx

```diff
@@ -6,4 +6,8 @@ export interface ClientOnlyProps {
   fallback?: ReactNode
 }
 
+/**
+ * This component ensures its children are only rendered on the client-side.
+ * @see {@link https://suspensive.org/docs/react/ClientOnly Suspensive Docs}
```

Comment:
Which one do you prefer between `https://suspensive.org/docs` and `https://suspensive.org/en/docs`

Reply:
https://suspensive.org/docs because https://suspensive.org/docs will automatically redirect user's page to proper language page if user use any language

Reply:
It would be good to standardize it to links without 'en' since they're not unified!

Reply:
Oh,, I merge this Pull Request. could you do that in next pull request please?

Reply:
Yes, I'll raise a PR to standardize the jsDoc overall!